### PR TITLE
ISSUE #1979 Metrics link missing

### LINKED ISSUE #1979: Broken "metrics" link in documentation
**BUG REPORT**

***Describe the bug***

The [`metrics.md`](https://github.com/apache/bookkeeper/blob/28dee8464764b0edceeb31ce24424f0947b5789b/site/docs/4.7.1/admin/metrics.md) file has a broken link pointing to an unknown site:
> For a full listing of available metrics, see the [Metrics](https://github.com/apache/bookkeeper/blob/28dee8464764b0edceeb31ce24424f0947b5789b/site/docs/reference/metrics) reference doc. <- (broken link)

This problem is also visible in the documentation page: https://bookkeeper.apache.org/docs/4.7.3/reference/metrics

***To Reproduce***

Just click the link posted above, you should see the problem.

***Expected behavior***

A "Metrics" page is expected with all the metrics that Bookkeeper exposes.

***Screenshots***

n/a.

***Additional context***

n/a.

### PR DESCRIPTION
**Pull request by sijia-w**
Fixes #1979 

The link for `Metrics` is broken. However, the `Metrics` file is empty. Therefore, I temporarily removed the link in all versions. The same problem happened to `configuration parameters`.

I closed pr #2386 as it brought changes from the master instead of a new branch.

### CODE REVIEW

File: site/docs/4.10.0/admin/metrics.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Provider | Provider class name
 
 ## Enabling stats providers in bookies
 
-There are two stats-related [configuration parameters](../../reference/config#statistics) available for bookies:
+There are two stats-related [configuration parameters](../../reference/config/) available for bookies:
```
Comment (Jennifer88huang) [inline site/docs/4.10.0/admin/metrics.md:25]:
```suggestion
Two stats-related [configuration parameters](../../reference/config/) parameters are available for bookies:
```

Comment (Jennifer88huang) [reply site/docs/4.10.0/admin/metrics.md:25]: Avoid using "there be" in technical writing. 
See IBM style guides.

File: site/docs/4.10.0/admin/metrics.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Metric collection
 
 BookKeeper enables metrics collection through a variety of [stats providers](#stats-providers).
 
-> For a full listing of available metrics, see the [Metrics](../../reference/metrics) reference doc.
+> For a full listing of available metrics, see the Metrics reference doc.
```
Comment (Jennifer88huang) [inline site/docs/4.10.0/admin/metrics.md:7]: Since the Metrics reference has no content at all, I suggest that we just remove this sentence.

**Comment by sijia-w** [comment]
@Jennifer88huang Thanks very much for your comments. Updated.

**Review by eolivelli — APPROVED** [review]
LGTM
thank you

**Pull request merged**